lib/doWhilst.js

```diff
@@ -19,7 +20,7 @@ import onlyOnce from './internal/onlyOnce';
  * passes. The function is passed a `callback(err)`, which must be called once
  * it has completed with an optional `err` argument. Invoked with (callback).
  * @param {Function} test - synchronous truth test to perform after each
```

Comment:
I really don't like how the `@typedef`s look in our code. I'd rather just link to MDN